drivers/include/net/netdev/ieee802154.h

```diff
@@ -121,6 +122,50 @@ typedef struct {
  */
 typedef struct netdev_radio_rx_info netdev_ieee802154_rx_info_t;
 
+/**
+ * @brief   Data structure for communicating IEEE 802.15.4 send and receive
+ *          call data to upper network layers.
+ */
+typedef struct {
+    uint8_t src[IEEE802154_LONG_ADDRESS_LEN]; /**< Source address */
+    uint8_t dst[IEEE802154_LONG_ADDRESS_LEN]; /**< Destination address */
+    uint8_t src_len;                          /**< Source address length */
+    uint8_t dst_len;                          /**< Destination address length */
+} netdev_ieee802154_data_hdr_t;
+
+/**
+ * @brief   Receive function for netdev IEEE 802.15.4 devices.
+ *
+ * Reformats the ieee802.15.4 mac frame to a generic
+ * @ref netdev_ieee802154_mac_t header using the frame content.
+ *
+ * @param[in]   dev     network device descriptor
+ * @param[out]  buf     buffer to write into or NULL
+ * @param[in]   len     maximum number of bytes to read
+ * @param[out] info     status information for the received packet. Might
```

Comment:
Okay, since you decided for left aligned below, this one needs adaption as well ;-)

Reply:
Done! :)